airflow/windows_extensions/pty.py

```diff
@@ -0,0 +1,170 @@
+"""Pseudo terminal utilities."""
+
+# Bugs: No signal handling.  Doesn't set slave termios and window size.
+#       Only tested on Linux.
+# See:  W. Richard Stevens. 1992.  Advanced Programming in the
+#       UNIX Environment.  Chapter 19.
+# Author: Steen Lumholt -- with additions by Guido.
```

Comment:
Where did this file come from? What is it licensed under?

Reply:
This file is copied from the installation directory of Python. Under Windows this is "...\Python37\Lib\pty.py". The license should be this one [https://docs.python.org/3/license.html](https://docs.python.org/3/license.html). We probably put it there to have all POSIX replacements in the same place.